music21/musicxml/test_m21ToXml.py

```diff
@@ -458,6 +458,63 @@ def testOutOfBoundsExpressionDoesNotCreateForward(self):
             int(tree.findall('.//direction/offset')[0].text),
             defaults.divisionsPerQuarter)
 
+    def testArpeggios(self):
+        expectedResults = (
+            'arpeggiate',
+            'arpeggiate',
+            'arpeggiate',
+            'arpeggiate up',
+            'arpeggiate up',
+            'arpeggiate up',
+            'arpeggiate',
+            'arpeggiate',
+            'arpeggiate',
+            'arpeggiate down',
+            'arpeggiate down',
+            'arpeggiate down',
+            'arpeggiate',
+            'arpeggiate',
+            'arpeggiate',
+            'non-arpeggiate bottom',
+            '',
+            'non-arpeggiate top',
+            'arpeggiate',
+            'arpeggiate',
+            'arpeggiate',
+        )
+        s = converter.parse(testPrimitive.arpeggio32d)
+        x = self.getET(s)
+        # helpers.dump(x)
+        mxPart = x.find('part')
+        mxMeasure = mxPart.find('measure')
+        for i, mxNote in enumerate(mxMeasure.findall('note')):
```

Comment:
Thanks for the test. I'm just a little worried about debugging it if it fails, say with musicxml 5.0 someday. Can you indent one more time under `with self.subTest(note_index=i):` so that your variable `i` is in the failure message if there's a failure?

Reply:
Looked around a bit to see how the heck _that_ works.  Got it.